surround/tests/cli/project_cli/Test_surround_init_lint.py

```diff
@@ -0,0 +1,117 @@
+import unittest
```

Comment:
The name of this file should reflect what is in the file, so something like `test_surround_init.py`